crates/symbolicator/src/services/shared_cache.rs

```diff
@@ -88,7 +72,7 @@ impl GcsState {
         let request = self
             .client
             .get(url)
-            .header("authorization", token.bearer_token())
+            .header("authorization", format!("Bearer {}", token.as_str()))
```

Comment:
There is btw https://docs.rs/reqwest/latest/reqwest/struct.RequestBuilder.html#method.bearer_auth
It does its own `format` internally though.